tutorials/nservicebus-step-by-step/4-publishing-events/tutorial.md

```diff
@@ -1,33 +1,32 @@
 ---
 title: "NServiceBus Step-by-step: Publishing Events"
-reviewed: 2023-11-24
+reviewed: 2024-05-21
 summary: In this 25-30 minute tutorial, you'll learn how to use the Publish/Subscribe pattern in NServiceBus to define, publish, and subscribe to an event.
 redirects:
 - tutorials/intro-to-nservicebus/4-publishing-events
 - tutorials/nservicebus-101/lesson-4
 extensions:
 - !!tutorial
-  nextText: "Next Lesson: Retrying errors"
+  nextText: "Next: Retrying errors"
   nextUrl: tutorials/nservicebus-step-by-step/5-retrying-errors
 ---
 
-So far in this tutorial, we have only sent **commands** – one-way messages from a sender to a specific receiver. There's another type of message we have yet to cover called an **event**. In many ways events are just like commands. They're simple classes and you deal with them in much the same way. But from an architectural standpoint, commands and events are *polar opposites*. This creates a useful dichotomy. We can take advantage of the properties of events to open up new possibilities in how we design software.
+So far in this tutorial, we have only sent **commands**, meaning one-way messages from a sender to a specific receiver. There's another type of message we have yet to cover called an **event**. In many ways events are just like commands. They're simple classes and you deal with them in much the same way. But from an architectural standpoint, commands and events are *polar opposites*. This creates a useful dichotomy. We can take advantage of the properties of events to open up new possibilities in how we design software.
```

Comment:
I feel like the original with the em-dash might read better in this case? But that's just preference.

Regardless,

```suggestion
So far in this tutorial, we have only sent **commands**, meaning one-way messages from a sender to a specific receiver. There's another type of message we have yet to cover called an **event**. In many ways, events are just like commands. They're simple classes and you deal with them in much the same way. But from an architectural standpoint, commands and events are *polar opposites*. This creates a useful dichotomy. We can take advantage of the properties of events to open up new possibilities in how we design software.
```